### ISSUE TITLE
Errno: 111

### ISSUE THREAD

**Issue by emirbyte** [deathsec/instagram-py#51]
**Select your issue type:** (check at least one)
- [x ] Bug
- [x] Question
- [ ] Suggestion
- [ ] Other (please describe):  

**Describe your issue:**
fatal error:: Tor configuration invalid or server down :: [Errno 111] Connection refused
I have my tor opened socks installed uncommented controlport but its not working.
**Ways to Reproduce the issue (optional):**

**Comment by SrieOffcl** [comment]
yeah .... the same problem for me too ..! what the solution for solve the fatal error ::
